enhancements/console/customize-catalogs.md

```diff
@@ -0,0 +1,184 @@
+---
+title: customize-catalogs
+authors:
+  - "@lokanandaprabhu"
+reviewers:
+  - "@jerolimov"
+  - "@spadgett"
+approvers:
+  - "@spadgett"
+  - "@cvogt"
+api-approvers:
+  - "@jerolimov"
+  - "@cvogt"
+creation-date: 2022-07-27
+last-updated: 2022-08-03
+tracking-link: 
+  - https://issues.redhat.com/browse/ODC-6695
+status: implementable
+---
+
+# Customize Catalogs
+
+## Summary
+
+This document provides details about how cluster administrator 
+can hide developer catalog and sub-catalogs. 
+
+## Motivation
+
+Cluster administrator wants to disable developer catalog or sub-catalogs for all users.
```

Comment:
This is more "what" than "why". Do we have more details about why it is desirable to be able to hide catalogs from users?

Reply:
The main reason is that customers don't want to use and support samples, helm charts, or whatever the extendable catalog provides. Support means here that they have an own team (infra, SRE, ...) that helps other (dev) teams to use the OpenShift console in an aligned way.